src/core/src/Context.cpp

```diff
@@ -90,10 +94,20 @@ void Context::initialize(int64_t contextId, const std::filesystem::path& cesiumE
     _session->resume();
 
     Cesium3DTilesSelection::registerAllTileContentTypes();
+
+    CESIUM_TRACE_INIT((cesiumExtensionLocation /
+                       fmt::format(
+                           "cesium-trace-{}.json",
+                           std::chrono::time_point_cast<std::chrono::microseconds>(std::chrono::steady_clock::now())
+                               .time_since_epoch()
+                               .count()))
+                          .string());
```

Comment:
Alternatively this could be written as:

```c++
#if CESIUM_TRACING_ENABLED
    const auto timestamp = std::chrono::time_point_cast<std::chrono::microseconds>(std::chrono::steady_clock::now())
                               .time_since_epoch()
                               .count();
    const auto path = cesiumExtensionLocation / fmt::format("cesium-trace-{}.json", timestamp);
    CESIUM_TRACE_INIT(path.string());
#endif
```

Both look kind of ugly...

Reply:
I agree both are ugly, but your alternative option is more readable to me. Mind swapping to it?

Reply:
I landed on something more compact:

```c++
#ifdef CESIUM_TRACING_ENABLED
    const auto timeNow = std::chrono::time_point_cast<std::chrono::microseconds>(std::chrono::steady_clock::now());
    const auto timeSinceEpoch = timeNow.time_since_epoch().count();
    const auto path = cesiumExtensionLocation / fmt::format("cesium-trace-{}.json", timeSinceEpoch);
    CESIUM_TRACE_INIT(path.string());
#endif
```